src/pkg/backup/details/details.go

```diff
@@ -35,10 +36,12 @@ type DetailsModel struct {
 // Print writes the DetailModel Entries to StdOut, in the format
 // requested by the caller.
 func (dm DetailsModel) PrintEntries(ctx context.Context) {
+	sl := dm.FilterMetaFiles()
```

Comment:
We should do this filter in OneDrive before we sent the `DetailsModel` to `PrintEntries` as it will otherwise affect others like Exchange which does not even have the idea of `.meta` files.